lib/menu.rb

```diff
@@ -0,0 +1,14 @@
+class Menu
+
+  def initialize
+    @menu = [{dish: 'BBQ',       price: 10},
+             {dish: 'Hamburger', price: 8},
+             {dish: 'Pizza',     price: 9},
+             {dish: 'Pasta',     price: 7},
+             {dish: 'Sushi',     price: 6}]
+  end
+
+  def show
```

Comment:
Style/TrivialAccessors: Use `attr_reader` to define trivial reader methods. (https://github.com/bbatsov/ruby-style-guide#attr_family)